[TLV-2021] added sponsors and removed sponsor link

Rommmmm opened the pull request:
added 3 sponsors and removed sponsor link from welcome page

glasnt commented:
There's some issues with this PR: 

 * You're changing Cloudinary without following the [sponsor logo update](https://github.com/devopsdays/devopsdays-web/blob/main/utilities/README.md#updating-a-sponsor-image-or-data) process
 * You're reassigning Velocity from O'Reilly Velocity Conference to another company called Velocity. Could you please create a new sponsor, probably `velocitytech`, and not reassign an existing sponsor ID. 
 
Thanks!

mattstratton reviewed (changes requested):
Per @glasnt comments - for the changed logo/image for cloudinary, please follow the script/process for updating a sponsor that is used elsewhere

secondly, for velocity, you need to make a new sponsor altogether, and not overwrite the existing sponsor.

Rommmmm commented:
Hey @glasnt and @mattstratton 
thanks for your comments, i fixed everything.
1. I added back O'Reilly Velocity and changed the new sponsor to be velocitytech
2. I used the utility to change Cloudinary pic to a new one 
Hope its ok now
